platform/darwin/src/NSBundle+MLNAdditions.mm

```diff
@@ -9,6 +9,7 @@ @implementation NSBundle (MLNAdditions)
 + (instancetype)mgl_frameworkBundle {
     NSBundle *bundle = [self bundleForClass:[MLNSettings class]];
 
+#ifndef MLN_CUSTOM_COMBINED_BUNDLE
     if (![bundle.infoDictionary[@"CFBundlePackageType"] isEqualToString:@"FMWK"]) {
```

Comment:
Since we are touching this code, could you replace "Mapbox" with "MapLibre"?

Reply:
Do we want to rename the expected bundle name too, or just the assertion for now?

Reply:
Just the message is OK, although I don't see the harm in changing the bundle name too.

It's the `apple_resource_bundle` in `platform/os/BUILD.bazel` it looks like.